### ISSUE TITLE
Is the training procedure result normal? Masked regions do not improve and appear to be random noise. 

### ISSUE THREAD

**Issue by junzhin** [facebookresearch/mae#190]
Hello,

thank you for the great work and the great repo.

I attempted to reproduce the pre-training of MAE-ViT-Large, and performed 68 epochs on a chest x -rays dataset with about 300 k medical images, and the loss stopped improving when it reached around 0.0045 loss without pixlossnorm. Additionally, the reconstruction results fail to predict the masked regions correctly. 

![image](https://github.com/facebookresearch/mae/assets/69834865/fbe35678-32c6-477f-83b6-dfeaf1e5840e)

Could you suggest a reason for that? Any idea why this is the case?

**Comment by CristoJV** [comment]
Hi @junzhin,

I'm facing the same errors. ![output](https://github.com/facebookresearch/mae/assets/44086220/1e1b1533-f400-4a12-898c-bd7966feae1a)
Pretrained on 500k faces for 20 epochs, resuming from the authors' pretrained version MAE-ViT-Base.

Did you manage to solve them?

EDIT: I fixed it by disabling the pixlossnorm.

Thank you!

**Comment by YuanXun2024** [comment]
Hi @CristoJV,

Can you share how to disable the pixlossnorm? I met the same problem. 

Thank you!

**Comment by ats4869** [comment]
I would like to ask how you use finetune to train the reconstruction model on your own data set. I see that through main_finetune.py only models for classification tasks can be generated.
